### PR TITLE
[KYUUBI #877] Add kubernetes configmap yaml file

### PR DESCRIPTION
**Pull request by timothy65535**
### _Why are the changes needed?_

Add kubernetes configmap yaml file
```
kubectl apply -f docker/kyuubi-configmap.yaml
```

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docker/kyuubi-pod.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 # Usage:
 # ConfigMap:
-#   kubectl create cm kyuubi-defaults --from-file=${KYUUBI_HOME}/conf/kyuubi-defaults.conf
```
Comment (yaooqinn) [inline docker/kyuubi-pod.yaml:20]: can we still keep it？

Comment (timothy65535) [reply docker/kyuubi-pod.yaml:20]: yeah

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
LGTM

File: docker/kyuubi-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    ## Kyuubi Configurations
+
+    #
+    # kyuubi.authentication           NONE
+    # kyuubi.frontend.bind.host       localhost
+    # kyuubi.frontend.bind.port       10009
+    #
+
+    # Details in https://kyuubi.readthedocs.io/en/latest/deployment/settings.html
```
Comment (pan3793) [inline docker/kyuubi-configmap.yaml:52]: insert new line

File: docker/kyuubi-configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Usage:
+#   kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/kyuubi-configmap.yaml
```
Comment (pan3793) [inline docker/kyuubi-configmap.yaml:19]: would you please also change Usage to 
```
LOCAL:
  ...
DEMO:
  ...
```

**Review by pan3793 — APPROVED** [review]

File: docker/kyuubi-configmap.yaml (hunk 3 above)
Comment (timothy65535) [reply docker/kyuubi-configmap.yaml]: Thanks for the suggestion，patch is updated accordingly.

**Comment by timothy65535** [comment]
@pan3793 Thanks for the suggestion, patch is updated accordingly.

**Comment by pan3793** [comment]
Thanks, merging for v1.3.0

**Pull request closed without merging**